timer/plugin.js

```diff
@@ -38,18 +39,35 @@ export default class CCTimer extends Plugin {
 		Utils.log('[timer] Connected to livesplit');
 		Utils.log('[timer] Loading config..');
 
-		const config = new Config(this.mod);
-		await config.load('settings.json');
+		const configs = [];
 
-		Utils.log('[timer] Loaded config: ', config);
+		//Global Split Settings
+		const mainConfig = new Config(this.mod);
+		await mainConfig.load('settings.json');
+		Utils.log('[timer] Loaded main config: ', mainConfig);
+		configs.push(mainConfig);
 
-		if (config.isIGT) {
+		//Optional Additional Autosplitters (ex. for segments)
+		const fs = require('fs');
+		const configFiles = fs.readdirSync(this.mod.baseDirectory + this.splitsDir);
```

Comment:
If that directory does not exist it will throw an error. You can use `fs.existsSync` to fix that.